Week3/app.js

```diff
@@ -0,0 +1,104 @@
+{
+    /*
+     * Q1.
+     * Write a program which would sort the below array in ascending order by name.
+     * Hint: String.prototype.localeCompare()
+     */
+    const people = [{
+        name: 'Richie Baros'
+    }, {
+        name: 'Jayson Karner'
+    }, {
+        name: 'Dylan Wanner'
+    }, {
+        name: 'Travis Tarwater'
+    }, {
+        name: 'Angelique Lyall'
+    }, {
+        name: 'Stacie Dyar'
+    }];
+
+
+    people.sort(function (a, b) {
+        return a.name.localeCompare(b.name);
+    });
+    console.log(people);
+
+
+    /*
+     * Q2.
+     * Write a function that accepts an array and returns only the even numbers in it.
+     * For example, if the input array is [13, 1, 4, 6, 8, 3, 2, 2], the output array should be [4, 6, 8, 2, 2].
+     */
+
+    const myArray = [13, 1, 4, 6, 8, 3, 2, 2];
+    let evenArray = [];
+    for (let i = 0; i < myArray.length; i++) {
+        if (myArray[i] % 2 === 0) {
+            evenArray.push(myArray[i]);
+        }
+    }
+    console.log(evenArray);
```

Comment:
1. Ah. The logic is correct but you missed the part where it is asked to _create a function_.
2. You can also use `forEach` in this case as we need to iterate over the entire array. `forEach` isn't useful if we need to break out of a loop based on some condition in which case a for loop or for-of loop makes more sense. 🙂

Reply:
I have created a function 👍